app-backend/api/src/main/scala/scene/Routes.scala

```diff
@@ -396,4 +403,34 @@ trait SceneRoutes
       }
     }
   }
+
+  def getSentinelMetadata(sceneId: UUID, metadataUrl: String): Route =
+    authenticate { user =>
+      authorizeAsync {
+        val authorizedIO = for {
+          auth <- SceneDao.authorized(user,
+                                      ObjectType.Scene,
+                                      sceneId,
+                                      ActionType.View)
+          datasource <- SceneDao.getSceneDatasource(sceneId)
+        } yield {
+          auth && datasource.id == UUID.fromString(
+            "4a50cb75-815d-4fe5-8bc1-144729ce5b42")
```

Comment:
Do we have this value in a config anywhere? If so it'd be useful to refer to it as a value instead of as a string